### DISCUSSION TITLE
How to export budibase app to another websites?

### DISCUSSION THREAD

**Discussion by senshi74** [Budibase/budibase#6875] [Help] [answered]
Hi, I just created a budibase app, and I would like to output them into an actual files app to use in different places, such as Shopify or random websites.

I might have missed the information on how to do them, so if anyone has, please let me know. Thanks in advance.

**Comment by mike12345567** [comment, marked as answer]
Hi @senshi74 - Budibase apps cannot be run independently of the Budibase stack, you can export apps from the Budibase Cloud and then run them in a self hosted instance of Budibase, but you cannot run the apps without the Budibase backend services.

Budibase apps require the full backend services as they have data integrations, automations etc built in which can only be performed on the backend.

**Reply by senshi74** [reply]
Hi @mike12345567 , thank you for the answer.
I might be very new to the concept of app system so I quite not fully understand from your pov. So the app itself can't be able to run without the budibase backend system itself, then how can we actually utilitize the budibase app? As well as asking the way to export the apps. I might try to do the self hosting from the docker and will it able to deploy to others user can use the app instead ?
Sorry if the question might be confusing, english was not my first language, any help will be very helpful.
Thanks.

**Reply by mike12345567** [reply]
Hi @senshi74, no problem, I'll try to explain to concept.

Basically, Budibase is your full app stack, you build in it, you store your data through it and you have your backend automations (think Zapier, N8N, etc) with it. The apps you build in Budibase are meant to be a lot more than the frontend, so we have a lot of services around that.

Once you've finished building an app in Budibase, you just click the "Publish" button in the top right of the builder, which will put your app into production. At that point, you can go to the users management screen from the portal (docs [here](https://docs.budibase.com/docs/adding-users)) and then assign them to the app. They will then be able to open Budibase and that app will show up in their portal.

The apps aren't designed to be used independently of Budibase due to our goal being around internal tooling, your users should be able to go to one central location and access all of the Budibase apps that you've built and published!

**Reply by senshi74** [reply]
Hi @mike12345567, thank you for your explanation. Is there any option for a custom link so people can auto-join the user section? I try to utilise it to be able to use it for some other plan.

**Reply by mike12345567** [reply]
Hi @senshi74 currently there are no mechanisms to allow automatic registration of users, users must be invited by an admin to the platform. In the future there will be other ways to handle invitations using user groups, as well as hopefully a mechanism to import a pool of users from a SSO provider, but these features are not yet available.
